etc/base.yaml

```diff
@@ -294,11 +314,6 @@ radar_rest_sources_backend:
   _chart_version: 1.2.3
   _extra_timeout: 0
   replicaCount: 1
-  postgres:
```

Comment:
I think it's good to instead of deleting these values to move them to a file in the mod directory and and documentation on how to use the RADAR-Kubernetes stack while using a managed PostgreSQL server such as RDS.

Reply:
Sounds good, but perhaps nice to add as a Issue, rather than an addition to this PR?